packages/joint-core/src/dia/Paper.mjs

```diff
@@ -1791,7 +1854,16 @@ export const Paper = View.extend({
             return this.model.getBBox() || new Rect();
         }
 
-        return V(this.cells).getBBox();
+        const cellLayerViews = Object.values(this._cellLayerViews);
```

Comment:
Can we use `Rect.fromRectUnion()` for simplicity here?
Also, please add a comment what is being done here.